test/functional/provider/admin/messages/inbox_controller_test.rb

```diff
@@ -48,4 +48,24 @@ def test_index_system_message
     assert_select 'title', "Inbox - Index | Red Hat 3scale API Management"
     assert_select '#export-to-csv', false, 'Export all Messages'
   end
+
+  test 'Prevent sending reply without subject' do
+    login_as(@admin)
+    reply = @message.reply
+    reply.update_attributes(body: 'Reply Body', subject: nil)
+
+    MessageWorker.drain
+    assert msg = Message.last
```

Comment:
why this assert?

Reply:
Okay may be we can remove assert and just keep msg